cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/InitCommand.java

```diff
@@ -119,15 +119,6 @@ public void execute() {
         String packageName = Optional.of(this.userDir.getFileName()).get().toString();
         if (argList != null && argList.size() > 0) {
             packageName = argList.get(0);
-            if (!ProjectUtils.validatePkgName(packageName)) {
```

Comment:
If I am not wrong, this the validation we do when the user provides a package name to the init command. i.e. `ballerina init <packageName>`. If so, we should keep this validation.

Reply:
We should only be guessing the package name if the user doesnt provide a package name. i.e `ballerina init`